app/commands/help/__init__.py

```diff
@@ -0,0 +1,11 @@
+from app.commands import Command
+
+
+class HelpCommand(Command):
+    def __init__(self, command_handler):
+        self.command_handler = command_handler
+
+    def execute(self):
+        print("Available commands:")
+        for command_name in self.command_handler.commands.keys():
+            print(f"- {command_name}")
```

Comment:
We can also add a description property to each commands so when we list them, it shows a small description about it.